### DISCUSSION TITLE
How to use the database of the old version with the new one?

### DISCUSSION THREAD

**Discussion by reclus23** [firefly-iii/firefly-iii#8034] [General]
hi everybody,  i used fireflyIII with great satisfaction until 3 months ago. now i had a problem with nas docker manager and i had to install firefly again. I saw that the installation of the current version has a different folder setting from the version i was using. how can i replace the database of the new version by putting the old one so i have all the old data? i never exported the old data but i still have all the folders of the old version. i use firefly in docker on the synology nas

**Comment by JC5** [comment]
I'm sorry, I changed nothing the last three months that I know of. What do you mean exactly?

**Comment by reclus23** [comment]
hi JC5, how can i replace the firefly database created today by putting the database i was using months ago? thanks for your help

**Reply by JC5** [reply]
Point the database to the old volume, it should upgrade itself.

**Reply by reclus23** [reply]
Hi JC5 thanks so much for the reply. I'm not an expert and I wouldn't want to damage the program which works very well. There is a way to take the file containing the data from the old database (which file would that be?) and open it with some program (which one?). I prefer to copy everything down rather than risk disrupting the program's functioning. many thanks

**Reply by JC5** [reply]
Sorry, I know too little about Synology to do that risk free. But if you have the raw volume files you can always copy them over the current files and see what happens.

**Reply by reclus23** [reply]
JC5 hi, sorry for the stupid question, but which files should I try to overwrite? many thanks

**Reply by JC5** [reply]
That's OK, but honestly I know very little about Synology as well.
